src/en/clouds-maas.md

```diff
@@ -4,89 +4,114 @@ Title: Using MAAS with Juju
 # Using a MAAS cloud
 
 Juju works closely with [MAAS][maas-site] to deliver the same experience
-on bare metal that you would get using any other cloud.
+on bare metal that you would get by using any other cloud.
 
-## Registering a MAAS cloud with Juju
+## Register a MAAS cloud with Juju
 
-Using the Juju `add-cloud` command, it is easy to add your MAAS clouds to
-Juju's list of known clouds. The command is interactive, and will ask for
-a name and the endpoint to use. A sample session is shown below.
-
-Running...
+Use the interactive `add-cloud` command to add your MAAS to Juju's list of
+clouds:
 
 ```bash
 juju add-cloud
 ```
-...will enter the interactive mode. Enter the desired values to continue.
 
-```
+Example user session:
+
+```no-highlight
 Cloud Types
- maas
- manual
- openstack
- vsphere
+  maas
+  manual
+  openstack
+  oracle
+  vsphere
 
 Select cloud type: maas
 
+Enter a name for your maas cloud: maas-cloud
 
-Enter a name for your maas cloud: mainmaas
-
-Enter the API endpoint url: http://maas.example.org:5240/MAAS
+Enter the API endpoint url: http://10.55.60.29:5240/MAAS
 
-Cloud "mainmaas" successfully added
-You may bootstrap with 'juju bootstrap mainmaas'
+Cloud "maas-cloud" successfully added
+You may bootstrap with 'juju bootstrap maas-cloud'
 ```
 
-This will add both the 'mainmaas' cloud, which you can confirm
-by running:
+We've called the new cloud 'maas-cloud' and used an endpoint of
+'http://10.55.60.29:5240/MAAS'.
+
+Now confirm the successful addition of the cloud:
 
 ```bash
 juju clouds
 ```
 
-This will list the newly added clouds, preceded with the prefix 'local:' which
-denotes that these are local clouds added by the user:
+Here is a partial output:
 
 ```no-highlight
-Cloud        Regions  Default        Type        Description
-aws               11  us-east-1      ec2         Amazon Web Services
-...
-mainmaas            0                 maas        Metal As A Service
+Cloud        Regions  Default          Type        Description
+.
+.
+.
+maas-cloud         0                   maas        Metal As A Service
 ```
 
-Before you bootstrap this cloud, it is necessary to add the relevant
```

Comment:
I think what you're saying is that because `add-cloud` is interactive the user may presume that it does everything needed to use a cloud. I'll add it back.